dspace-api/src/main/java/org/dspace/browse/BrowseEngine.java

```diff
@@ -684,13 +687,13 @@ private int getTotalResults(boolean distinct)
         // our count, storing them locally to reinstate later
         String focusField = dao.getJumpToField();
         String focusValue = dao.getJumpToValue();
-        String orderField = dao.getOrderField();
+        // String orderField = dao.getOrderField();
```

Comment:
Again, it looks like you've commented out code here, and on line 696 and 706.  Should these be removed?  It looks (to me) like the ordering may not be necessary in this `getTotalResults()` method, as it doesn't matter what order the results come back in.